pyproject.toml

```diff
@@ -68,7 +67,13 @@ build = "cp39-*64 cp310-*64 cp311-*64 cp312-*64"
 skip = "*musllinux*"
 
 test-command = "python -m pytest {project}/tests"
-test-requires = ["pytest", "pandas>=2.0.0", "polars~=1.2.0", "narwhals"]
+test-requires = [
+  "pytest",
+  "pandas>=2.0.0",
+  "polars~=1.2.0",
+  "narwhals",
+  "tableauhyperapi",
```

Comment:
Might also want to come up with a way to monkeypatch the tableau installation not existing, and making sure the pantab runtime works as expected